Idea.js

```diff
@@ -0,0 +1,17 @@
+class Idea {
+  constructor(idea){
```

Comment:
I agree it is a good call- at least at our present understanding -to feed the constructor an object and not individual parameters. Strikes me as more dynamic/efficient down the road.